packages/matchbox/src/components/TextField/TextField.js

```diff
@@ -138,6 +141,7 @@ const TextField = React.forwardRef(function TextField(props, userRef) {
     height: multiline ? 'auto' : '2.5rem',
     lineHeight: multiline ? '300' : '2.5rem',
     py: multiline ? '300' : '0',
+    hasError: !!error,
```

Comment:
Found a bug, we were never passing through `hasError` so the textfields were never red